test/parallel/test-fs-rm.js

```diff
@@ -280,3 +282,45 @@ function removeAsync(dir) {
     message: /^The value of "options\.maxRetries" is out of range\./
   });
 }
+
+{
+  // IBMi has a different access permission mechanism
+  // This test should not be run as `root`
+  if (!common.isIBMi && (common.isWindows || process.getuid() !== 0)) {
+    function makeDirectoryReadOnly(dir) {
+      let accessErrorCode = 'EACCES';
+      if (common.isWindows) {
+        accessErrorCode = 'EPERM';
+        execSync(`icacls ${dir} /deny "everyone:(OI)(CI)(DE,DC)"`);
```

Comment:
```suggestion
        execSync(`icacls ${dir} /deny "everyone:(OI)(CI)(D,DC)"`);
```

Reply:
Apparently `D` breaks the test, so I changed it back to DE